src/SignalR.Orleans/SignalrConfig.cs

```diff
@@ -37,13 +39,13 @@ public SignalrOrleansSiloConfigBuilder Configure(Action<ISiloBuilder, HostBuilde
 
 public class SignalrOrleansSiloHostConfigBuilder : SignalrOrleansConfigBaseBuilder
 {
-	internal Action<ISiloHostBuilder, HostBuilderConfig> ConfigureBuilder { get; set; }
+	internal Action<ISiloBuilder, HostBuilderConfig> ConfigureBuilder { get; set; }
```

Comment:
this class most probably can be removed as now its duplicated from above

Reply:
done